app/Http/Controllers/PetController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Pet;
+
+class PetController extends Controller
+{
+    
+    public static function show()
+    {
+        // controller actions typically return a view and a model
+        return view(
+            'pets', // name of the view, i.e. views/game-list.blade.php
+            [
+                //the keys define variables in the template, i.e. $games
+                'pets' => self::getPets(),
+            ]);
+    }
+
+    public static function getPets(): array {        
```

Comment:
this can be private scope since it's only called within the class definition

Reply:
Okay, I just changed it. Thanks!